tools/build/scripts/utils/addPackageMetadata.js

```diff
@@ -15,8 +15,14 @@ const rootPackageJson = process.env.RUSHSTACK_FILE_ERROR_BASE_FOLDER
 // Check if path to root package.json is valid.
 const rootPackageJsonPath = require.resolve(rootPackageJson);
 
+//Find the package.json of the project and retrieve the version and repository URL
+const packageJsonPath = path.join(process.cwd(), "package.json");
+const packageJson = JSON.parse(FS.readFileSync(packageJsonPath));
```

Comment:
do you need this new var packageJsonPath? wouldnt it work fine w/o it?

Reply:
You mean by just using the rootPackageJsonPath variable? I tried that at first, but it was getting the package.json present at the root of core and not for each project.

Reply:
gotcha; damn monorepos